vhs/vhs-test.js

```diff
@@ -0,0 +1,79 @@
+const { describe } = require("mocha");
+var {createTape, reset, createCollection, previewCollection} =  require("./vhs")
+var assert = require('chai').assert;
+
+describe("Tape", function() {
+
+  it.skip("should create tape", function() {
+    var genericTape = createTape("",false)
+   
+    assert.deepEqual(genericTape, {title:"", readyToPlay: false})
+  })
+
+  it.skip("should have a title", function(){
+    var littleMermaid = createTape("The Little Mermaid", false)
+    var landBeforeTime = createTape("The Land Before Time", false)
+
+    assert.equal(littleMermaid.title,"The Little Mermaid")
+    assert.equal(landBeforeTime.title,"The Land Before Time")
+  })
+
+  it.skip("should have a readyToPlay status by default", function() {
+    var littleMermaid = createTape("The Little Mermaid", false);
+    var landBeforeTime = createTape("The Land Before Time", false);
+
+    assert.equal(littleMermaid.readyToPlay, false);
+    assert.equal(landBeforeTime.readyToPlay, false);
+  })
+
+  it.skip("should be able to be readyToPlay", function() {
+    var fargo = createTape("Fargo", false);
+    var resetFargo = reset(fargo);
+    var betterOffDead = createTape("Better Off Dead",true)
+    var resetBetterOffDead = reset(betterOffDead)
+  
+    assert.equal(resetFargo.readyToPlay, true);
+    assert.equal(resetBetterOffDead.readyToPlay, true); 
+  })
+
+  it.skip("should not reset if it has already been reset", function() {
+    var betterOffDead = createTape("Better Off Dead",true)
+    var resetBetterOffDead = reset(betterOffDead)
+
+    assert.equal(resetBetterOffDead.readyToPlay, true);
+  })
+})
+
+describe("Collection", function(){
+
+  it.skip("should add tapes to collection", function(){
+    var fargo = createTape("Fargo",true)
+    var betterOffDead = createTape("Better Off Dead", false);
+   
+    var collection = createCollection(fargo, betterOffDead)
```

Comment:
I'm really fried, so please disregard this if it sounds wild 😂 

It looks like createCollection accepts 2 arguments here, but later (line 67) it accepts 3. Are students meant to learn about the arguments object or rest parameters at this point? Or maybe they are meant to write in a conditional to make the function behave differently for 2 or 3 arguments? Figured I'd ask to make sure we were all on the same page regarding goals for these tests.

Reply:
Good Catch! I meant to have this use three arguments. I'll update it soon :)